### PR TITLE
[native] Fix presto-native-execution README for running setup-adapters script

### PR DESCRIPTION
**Pull request by imjalpreet**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/README.md [hunk 1]
```diff
@@ -27,7 +27,15 @@ available inside `presto/presto-native-execution/scripts`.
...
 this folder. All the dependencies are installed in the system default location eg: `/usr/local`.
 
+On an intel macOS machine, you can install the dependencies using the following commands:
+```bash
+./scripts/setup-macos.sh
+```
 
+On an M1 macOS machine, you can install the dependencies using the following commands:
+```bash
+CPU_TARGET="arm64" INSTALL_PREFIX="/opt/homebrew" ./scripts/setup-macos.sh
```
Comment (majetideepak) [inline presto-native-execution/README.md:37]: We need to and can automate this. I will add a fix and we can avoid this update.

**Review by majetideepak — CHANGES_REQUESTED** [review]
@imjalpreet Most of the updates here should be handled by the setup scripts.
Can you open an issue? We should fix this.

**Comment by majetideepak** [comment]
> Should I revert the other document changes apart from that?

Let's fix the CPU_ARCH, INSTALL_PREFIX first. We can then document whatever is remaining. I will update here.
But yes, likely only the adapters.sh change is needed.

**Comment by majetideepak** [comment]
CPU_ARCH requirement should be fixed with this https://github.com/facebookincubator/velox/issues/5204

**Comment by majetideepak** [comment]
@imjalpreet feel free to keep the adapters.sh fix and revert others. We can get that merged.

**Comment by imjalpreet** [comment]
@majetideepak Updated the PR

**Review by majetideepak — APPROVED** [review]

**Pull request merged**